documentation/SA1135.md

```diff
@@ -0,0 +1,59 @@
+## SA1135
+
+<table>
+<tr>
+  <td>TypeName</td>
+  <td>SA1135UsingDirectivesMustBeQualified</td>
+</tr>
+<tr>
+  <td>CheckId</td>
+  <td>SA1135</td>
+</tr>
+<tr>
+  <td>Category</td>
+  <td>Readability Rules</td>
+</tr>
+</table>
+
+## Cause
+
+A using directive is not qualified.
+
+## Rule description
+
+ A violation of this rule occurs when a using directive is contained within a namespace and is not qualified.
```

Comment:
:bulb: It would be good to include documentation saying that references to simple type names do not need to be qualified. Only namespace references need to be qualified.

Reply:
I've updated the documentation to state that alias definitions to types within the same namespace do not need to be qualified. The current implementation does no checking against simple types, not sure it that is important enough to add, but please file an issue if it should be corrected.